src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageClientMiddleLayer.cs

```diff
@@ -72,7 +72,7 @@ public override async Task<JToken> HandleRequestAsync(string methodName, JToken
                 var token = JToken.FromObject(requestParams);
                 var result = await sendRequest(token).ConfigureAwait(false);
                 var edits = result?.ToObject<TextEdit[]>();
-                if (edits == null)
+                if (edits == null || edits.Length == 0)
```

Comment:
What's this resolve?

Reply:
Nothing. Just spotted an early exit opportunity.